app/views/layouts/_header.html.erb

```diff
@@ -31,7 +31,7 @@
         </li>
       </div>
       <li class="nav-item px-1">
-        <a class="nav-link navbar-item navbar-text <%= request.path == "/features" ? "active" : "" %>" href="/#home-features-section"><%= t("layout.link_to_features") %></a>
+        <a class="nav-link navbar-item navbar-text <%= request.path == "/features" ? "active" : "" %>" href="/#home-features-section" onclick="event.preventDefault(); window.scrollTo({ top: document.getElementById('home-features-section').offsetTop - 100, behavior: 'smooth' });"><%= t("layout.link_to_features") %></a>
```

Comment:
Not a correct approach to resolve this.

You should add `scroll-margin-top: 100px` instead.